flixel/system/FlxAssets.hx

```diff
@@ -28,7 +28,7 @@ class GraphicLogo extends BitmapData {}
 class GraphicVirtualInput extends BitmapData {}
 
 @:file("assets/images/ui/virtual-input.txt")
-class VirtualInputData extends ByteArray {}
+class VirtualInputData extends #if lime_legacy ByteArray #else ByteArrayData #end {}
```

Comment:
This breaks compilation with older OpenFL versions, so the only way to have this in here is to add a check for the OpenFL / Lime version (which is somewhat tricky with this change not being released yet).

Also seems strange that OpenFL would need to break compatibility here.